build.sbt

```diff
@@ -41,14 +41,28 @@ lazy val allVersionCompilerLintSwitches = Seq(
   "-feature",
   "-unchecked",
   "-Xfatal-warnings",
-  "-Xlint",
   "-Yno-adapted-args",
   "-Ywarn-dead-code"
 )
 
-lazy val newerCompilerLintSwitches = Seq(
+lazy val scala211Flags = Seq(
   "-Ywarn-unused-import", // Not available in 2.10
-  "-Ywarn-numeric-widen" // In 2.10 this produces a some strange spurious error
+  "-Ywarn-numeric-widen" // In 2.10 this produces a strange spurious error
+)
+
+// Scala 2.12.2 has excessive warnings about unused implicits
```

Comment:
Is this the issue that you're talking about here?
https://github.com/scala/bug/issues/10270
If so, would you be willing to add a link to it, so we can tell in the future if this problem gets fixed?